proxy/README.md

```diff
@@ -26,8 +26,8 @@ $ cp proxy.example.yml proxy.yml
 
 By default, the OpenLineage proxy uses the following ports:
 
-* TCP port `8080` is available for the HTTP API server.
-* TCP port `8081` is available for the admin interface.
+* TCP port `5000` is available for the HTTP API server.
+* TCP port `5001` is available for the admin interface.
```

Comment:
I would keep `8080` and  `8081` the as the default ports. When running the proxy server via:

```
./gradlew runShadow
```

The ports are `8080` and `8081`. These same default ports are defined in [`proxy.example.yml`](https://github.com/OpenLineage/OpenLineage/blob/main/proxy/proxy.example.yml#L6)

Reply:
Ok, got it.